### PR TITLE
Add Serbian translation

### PR DESCRIPTION
**Pull request by bingbinglee**
Serbian translation provided by Rados

### CODE REVIEW

**Comment by bingbinglee** [comment]
Whoops. No clue why the deploys are failing.

File: i18next-scanner.config.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
-const AVAILABLE_LANGS = ['en', 'hi', 'it']
+const AVAILABLE_LANGS = [
```
Comment (menxit) [inline i18next-scanner.config.js:1]: We should delete one of AVAILABLE_LANGS

File: public/locales/sr/translation.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	"Copy address": "Kopiraj adresu",
+	"Reset wallet": "Resetovanje novčanika",
+	"Clear browser storage and use another wallet": "Brisanje podataka sa pretraživača i korišćenje drugog novčanika",
+	"Logout": "Odjavljivanje",
+	"Profile": "Profil",
+	"Transactions": "Transakcije",
+	"Old users could use m/44'/60'/0'/0/0": "Stari korisnici mogu da koriste m/44'/60'/0'/0/0",
+	"Import": "Uvezi",
+	"Seed phrase": "Seed fraza",
+	"Address copied": "Adresa kopirana"
```
Comment (menxit) [inline public/locales/sr/translation.json:54]: We should add ledger strings:
```
"Connect Ledger": "Povežite knjigu",
"ledger_connect_confirmation": "Assicurati che l'app idena-ledger sia attualmente aperta e permetti a idena-pocket di leggere il tuo indirizzo."
```

File: i18next-scanner.config.js [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+	"en",
```
Comment (menxit) [inline i18next-scanner.config.js:2]: We should delete one of AVAILABLE_LANGS

File: public/locales/sr/translation.json (hunk 2 above)
Comment (menxit) [inline public/locales/sr/translation.json:54]: We should add ledger strings:
```
"Connect Ledger": "Povežite knjigu",
"ledger_connect_confirmation": "Assicurati che l'app idena-ledger sia attualmente aperta e permetti a idena-pocket di leggere il tuo indirizzo."
```

Comment (bingbinglee) [reply public/locales/sr/translation.json]: updated

**Pull request merged**